### PR TITLE
DIG-931: Vault aws policy needs update permissions

### PR DESCRIPTION
**Pull request by daisieh**
Also added in an env var to be passed in for htsget; it is optional so no change to test.

### CODE REVIEW

File: lib/htsget-server/docker-compose.yml [hunk 1]
```diff
@@ -44,6 +44,7 @@ services:
       HTSGET_TEST_KEY: "hoodlebug"
       HTSGET_URL: ${TYK_LOGIN_TARGET_URL}/${TYK_HTSGET_API_LISTEN_PATH}
       OPA_URL: ${OPA_PRIVATE_URL}
+      OPA_SITE_ADMIN_KEY: ${OPA_SITE_ADMIN_KEY} # only required if not equal to "site_admin"
```
Comment (shaikh-rashid) [inline lib/htsget-server/docker-compose.yml:47]: If this is a secret, it should be provided via `secret` and not `environment`

**Review by shaikh-rashid — CHANGES_REQUESTED** [review]
Request for clarification

File: lib/htsget-server/docker-compose.yml (hunk 1 above)
Comment (daisieh) [reply lib/htsget-server/docker-compose.yml:47]: It's not a secret, it's just the value "site_admin" usually, but I wanted to leave it open for a site to have a different role name if they choose.

**Review by shaikh-rashid — APPROVED** [review]
Reviewed and approved

**Pull request merged**